packages.json

```diff
@@ -1,5 +1,4 @@
-[
-  {
+[{
```

Comment:
Please don't make changes to the JSON itself :)

Reply:
Oh, it is vscode default format-on-save. I thought it will be auto re-format again when merge =p